pkg/webhooks/validation/controller/controller.go

```diff
@@ -79,8 +78,8 @@ func (o Options) Validate() error {
 	if o.ServiceName == "" || !labels.IsDNS1123Label(o.ServiceName) {
 		errs = multierror.Append(errs, fmt.Errorf("invalid service name: %q", o.ServiceName))
 	}
-	if o.CAPath == "" {
-		errs = multierror.Append(errs, errors.New("CA cert file not specified"))
+	if o.CABundleWatcher == nil {
```

Comment:
why do we have a watcher here but not for injector?

Reply:
I searched related code, and found there was no cabundle  watch before. Agree,  they should be same.